src/components/ArticlesByTopic.jsx

```diff
@@ -0,0 +1,26 @@
+import { useLocation } from "react-router-dom";
+import ArticleBlock from "./ArticleBlock";
+import ArticleBlockSmall from "./ArticleBlockSmall";
+
+function ArticlesByTopic({ articles }) {
```

Comment:
this is unrelated to either tickets 5 or 6 - Try not to add any additional code that isn't required to complete the task that this branch is referring to - so that we can keep all relevant code on the relevant branch :slightly_smiling_face: